### ISSUE TITLE
[Invoices] Create Customer for Nonprofit in Stripe

### ISSUE THREAD

**Issue by clatwell** [Good-Bids/goodbids#371]
Before we can create an Invoice for a Nonprofit in Stripe, we need to have a `customerId` associated with the Nonprofit site. In #369, we added the Stripe customer ID field to Nonprofits. For this ticket, we need to add the ability to create a Customer object for the Nonprofit with the GOODBIDS Stripe Account via the Stripe API. Two options for when this happens: 

1. Before we create the Invoice post in WP 
2. Before we send the Create Invoice post to Stripe 

If we're planning to store the NP Stripe Customer ID on the Invoice post, I think it makes sense to batch this with the action that creates the Invoice post in WP. But, if we're just planning to pull the Customer ID directly from the NP site to build the Create Invoice POST to Stripe, we can probably do this step with the Create Invoice post. I pulled this out as its own ticket bc I wasn't sure when it would happen :) 

Either way, we need to... 

- Check to see if there's a value in the Stripe Customer ID field for the Nonprofit in the site metadata table 
- If there's not, send a POST request to the [/v1/customers](https://stripe.com/docs/api/customers/create) Stripe endpoint with 
  - name = Nonprofit Site Name 
  - email = Site Admin email 
- The response will include an id for the Customer in Stripe 
- Store the Customer ID for the Nonprofit in the site metadata table
